app/subsystems/entity/api/create_role.rb

```
@@ -3,9 +3,9 @@ class Entity::CreateRole
 
   protected
 
-  def exec
-    role = Entity::Role.create
-    transfer_errors_from(role, {type: :verbatim}, true)
+  def exec(role_type = :unassigned)
+    role = Entity::Role.create(role_type: role_type)
```

Does calling `create!` remove the need to call `transfer_errors_from`, since any validation errors will be made into exceptions that lev no longer swallows.  This would be ok because we're not doing anything with user input here so any errors are going to be truly exceptional.